R/rope_range.R

```diff
@@ -63,7 +63,12 @@ rope_range.brmsfit <- function(x, ...) {
   information <- insight::model_info(x)
 
   if (insight::is_multivariate(x)) {
-    mapply(function(i, j) .rope_range(i, j), x, information, response)
+    ret <- mapply(function(i, j) .rope_range(x, i, j), information, response)
+
+    # return matrix as named list
+    # see https://stackoverflow.com/questions/6819804/how-to-convert-a-matrix-to-a-list-of-column-vectors-in-r
+    # this is not the fastest solution bot keeps columns names
+    as.list(as.data.frame(ret))
```

Comment:
Can you elaborate why this is necessary?

Reply:
Actually, this is not really necessary. However, I decided to do this to have more user friendly arguments.

The above code returns a matrix, a column for every dependent variable and two rows for the lower and upper ROPE limit. In the case that a user specifies own ROPE limits, they would need to create a matrix with these rows and columns. Most of the time, I find this cumbersome and error-prone because I need to create the matrix and pay attention to build the correct columns and rows. I thought it would be easier to use a list with named vectors which correspond to the dependent variables (comparable to `contrasts` argument in a `lm` function).

In addition, I wanted to keep the return value here and the arguments in the `rope` function consistent. So I decided to change the return value here. However, it would be possible to convert a user-specified list to a matrix as well ...

Reply:
ok, got it. thanks!